docs/CNVs/CNV-methods.tex

```diff
@@ -482,6 +497,135 @@ \subsection{Using Panel of Normals for Allelic Fraction Model} \label{allelic-Po
+\end{equation}
+%
+
+
+The M step for $\vm$ is routine and comes out to:
+\begin{align}
+\vm = \left( \sum_s \vPsi_{s}^{-1} \right)^{-1} \sum_s \left[ \vPsi_{s}^{-1}  (\vW  E[\vz_{s}] - \vm_{s} ) \right] 
+\end{align}
+
+The M step for $\vPsi_t$ is
+\begin{align}
+\sum_s \frac{1}{\vPsi_t + \Sigma_{st}} - \frac{\left( \vW_t  E[\vz_s \vz_s^T] \vW_t^T + (\vm_t - \vm_{st})^2 +2 (\vm_t - \vm_{st}) \vW_t E[\vz_s] \right)}{(\vPsi_t + \Sigma_{st})^2} = 0,
+\end{align}
+which is straightforward to solve numerically.  Alternatively, we could approximate $\vSigma_s \approx \bar{\vSigma} \equiv \frac{1}{S} \sum_s \vSigma_s$, in which case we obtain
+%
+\begin{equation}
+\vPsi_t + \bar{\vSigma}_t = \frac{1}{S} \sum_s \left( \vW_t  E[\vz_s \vz_s^T] \vW_t^T+ (\vm_t - \vm_{st})^2 +2 (\vm_t - \vm_{st})  \vW_t E[\vz_s] \right)
+\end{equation}
+%
+
+The M step equation for the row $\vW_t$ is also routine and gives
+%
+\begin{equation}
+\vW_t^T = \left( \sum_s \Psi_{st}^{-1}  E[\vz_{s} \vz_{s}^T]  \right)^{-1} \sum_s \Psi_{st}^{-1} ( \vm_{st} - \vm_t) E[\vz_{s}]
+\label{M_step_W}
+\end{equation}
+
+We may automatically select the number of latent features to use via an automatic relevance determination (ARD) prior on $\vW$.  That is, we add a term
+\begin{equation}
+P(\vW) = \prod_k \left( \frac{\alpha_k}{2 \pi} \right)^{T/2} \exp \left( -\frac{1}{2} \alpha_k \sum_t W_{tk}^2  \right)
+\end{equation}
+The E step is unaffected by this addition and the only change to the M step is that in Equation \ref{M_step_W} we now have
+\begin{equation}
+\vM_t = {\rm Diag}(\alpha_1, \alpha_2 \ldots \alpha_K) + \sum_s \Psi_{st}^{-1}  E[\vz_{s} \vz_{s}^T] 
+\end{equation}
+
+The ARD coefficients $\alpha$ are determined via the evidence approximation in which their marginal likelihood is maximized.  That is, we set
+\begin{align}
+\frac{\partial}{\partial \alpha_k} \int P(\vW | \alpha) {\rm likelihood}(\vW) \, d \vW =& 0 \\
+\frac{1}{2} \int \left( \frac{T}{\alpha_k} - \sum_t W_{tk}^2 \right) {\rm posterior}(\vW) \, d \vW =& 0 \\
+\end{align}
+If we posterior is tightly peaked about the MLE estimate $\vW$, this has the solution
+\begin{equation}
+\alpha_k = \frac{T}{\sum_t W_{tk}^2}
+\end{equation}
+The meaning of these parameters is that $\alpha_k \rightarrow \infty$ sets the $k$-th column of $\vW$ to zero, turning off the $k$-th latent feature.  Thus we can initially choose a liberal estimate of $K$ and the model will automatically become more parsimonious.
+
+We can apply the parameters learned from the panel of normals to single-sample calling, which requires the likelihood as a function of the copy numbers $\vc$.  Applying the same E step as above, the likelihood is
+\begin{equation}
+P({\rm data} | \vc) \propto \prod_t \exp \left[ -\frac{1}{2} \left(  (\vm_t - \vm_{st})^2 +2 (\vm_t - \vm_{st}) \vW_{t} E[\vz_{s}] \right)\Psi_{st}^{-1} \right],
```

Comment:
Do you suggest intertwining PON creation and CNV calling into a single workflow? i.e. estimate PON (W, \Sigma, m) by via EM (z latent, c_st a fixed hyper parameter), followed by estimating c_st by keeping the PON fixed (via HMM, CBS, maximum likelihood on Eq. 63, etc), and continue until convergence? I can think of two caveats:

(1) Perhaps it is going to be very time consuming compared to our current one-pass scheme (PON creation -> CNV calling)
(2) c_st is not treated quite as a latent variable, but as a hyper parameter determined by an auxiliary segmentation step. Do we have a guarantee that the combined iterations of PCA-EM and CNV calling actually converges? in other words, do we have a guarantee that P(n, z, c | ...) increases after each pass (as in the usual EM)? perhaps the answer is positive (e.g. if the CNV calling increases Eq. 63), but I can not quite see it.